homeassistant/components/conversation/default_agent.py

```diff
@@ -335,10 +355,32 @@ async def async_process(self, user_input: ConversationInput) -> ConversationResu
         assert lang_intents is not None
 
         # Slot values to pass to the intent
-        slots = {
-            entity.name: {"value": entity.value, "text": entity.text or entity.value}
-            for entity in result.entities_list
-        }
+        slots: dict[str, Any] = {}
+
+        # Automatically add device id
```

Comment:
Why is this necessary?

Reply:
We don't pass `device_id` into `intent.async_handle`. Maybe we should?

Reply:
Yes, but should it then be a new kwarg instead of merged into slots ?

Reply:
PR for this: https://github.com/home-assistant/core/pull/117442